lib/api.js

```diff
@@ -320,28 +323,28 @@ class Api extends Emittery {
 	}
 
 	async _computeForkExecArgv() {
-		const execArgv = this.options.testOnlyExecArgv || process.execArgv;
-		if (execArgv.length === 0) {
-			return Promise.resolve(execArgv);
-		}
+		const {nodeArguments: configExecArgs} = this.options;
 
-		// --inspect-brk is used in addition to --inspect to break on first line and wait
-		const inspectArgIndex = execArgv.findIndex(arg => /^--inspect(-brk)?($|=)/.test(arg));
-		if (inspectArgIndex === -1) {
-			return Promise.resolve(execArgv);
-		}
+		const mainProcessArgs = minimist(process.execArgv);
```

Comment:
Could we cache this? This function runs for every test file.

Reply:
Not sure, what I correctly understand you, check please